### ISSUE TITLE
Unify the mastr data to one consistent dataset

### ISSUE THREAD

**Issue by ulfmueller** [openego/powerd-data#10]
Due to time issues in the egon-data project the workflow works with two different mastr data sets. This has to be unified.

Mainly chp (most tough one to change). Check out what else has to be addressed.

Check out whether or how to check the mastr data accuracy with e.g. NEP power plant list (or leave it entirely to the validation).

**Comment by ClaraBuettner** [comment]
Check if #9 is done

**Comment by ClaraBuettner** [comment]
CHP plants now use the new MaStR data (see PR #24 )

**Comment by CarlosEpia** [comment]
Power plants are also working using the new MaStR data. Are there other tasks to be modified?

**Comment by ClaraBuettner** [comment]
> Power plants are also working using the new MaStR data. Are there other tasks to be modified?

Are you sure that all parts of the power plant data set use the new data? As far as I remember, there were some power plants where we didn't change it (hydro, biomass, conventional)?
e.g. here: https://github.com/openego/powerd-data/blob/5096b3f109b838b45848a55ef9ebe19a4e88e07f/src/egon/data/datasets/power_plants/__init__.py#L243-L245

**Comment by CarlosEpia** [comment]
Good point. Biomass is still using the old MaStR version. I will change it. Regarding hydro, it is already using the new version, as well as the conventional power plants (we read directly for Status2019 the CSV files).

**Comment by CarlosEpia** [comment]
Update: Generators for status2019 only use the new MasTR data. For eGon2035 some additional changes would be required. Merging from egon-data will help but still some adjustments are required.
